### PR TITLE
Deprecate legacy CustomAdNotification griffin studies

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

**Comment by tmancey** [comment]
https://github.com/brave/brave-variations/pull/165

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` as per https://github.com/brave/brave-variations/pull/165#issuecomment-974385629.

**Pull request merged**
